### DISCUSSION TITLE
MyGPTReader 部署到 Fly.io 教程完整记录

### DISCUSSION THREAD

**Discussion by Tanverkader** [madawei2699/myGPTReader#45] [Show and tell]
非常感谢大伟老师分享的这个项目，前几天我也尝试自己部署到了 Fly.io 。
我看 issue 里面也有人问怎么自己部署全部功能，就把过程记录了下来：
[MyGPTReader 部署到 Fly.io 教程完整记录(全部功能) - GPT4AllPro](https://gpt4all.pro/chatgpt-in-slack-mygptreader-deployment-to-fly-io-tutorial-complete-documentation-all-features/)

希望对大家有所帮助：)

**Comment by madawei2699** [comment]
好详细的部署方案，期待能提一个 PR 到主分支，其实最简单的就是把你的文章链接放到 readme 里即可。
